### ISSUE TITLE
WHAT is going on with this repo?

### ISSUE THREAD

**Issue by x-ili-x** [lllyasviel/stable-diffusion-webui-forge#2478]
Every time I try to work this repo it's a new problem. Fresh machines, fresh linux and windows installs, by the book rollbacks to try to find what&where it went wrong - it's just a mess, dude. WTF are we doing here?

**Comment by jswag245** [comment]
This repo is an experimental repo. The readme states Forge is also used to "study experimental features" and that "WebUI Forge is now under some constructions, and docs / UI / functionality may change with updates." 

That is what we are doing here. If you are having an issue with the inconsistency, I recommend you check out Automatic1111's Webui.

**Comment by x-ili-x** [comment]
No. $1000 to anyone who can fix flux from going full static whenever using highres fix and inpainting. You find me a fix, you get the money.

**Comment by jswag245** [comment]
Nice! If you're going to include a bounty, I highly recommend putting in a discussion post as well as a Reddit post on some subs for maximum exposure. Also, it would help to close this issue and make a new one with a relevant title and body text.

**Comment by x-ili-x** [comment]
copy your entire install to a drive where i can download it and verify, and yes, you'll get the money.
